pyvista/plotting/plotter.py

```diff
@@ -366,11 +367,11 @@ def suppress_rendering(self):  # numpydoc ignore=RT01
         return self._suppress_rendering
 
     @suppress_rendering.setter
-    def suppress_rendering(self, value):  # numpydoc ignore=GL08
+    def suppress_rendering(self, value) -> None:  # numpydoc ignore=GL08
         self._suppress_rendering = bool(value)
 
     @property
-    def render_window(self):  # numpydoc ignore=RT01
+    def render_window(self) -> _vtk.vtkRenderWindow | None:  # numpydoc ignore=RT01
```

Comment:
The `|` Union is a Python 3.10 feature.  Will we need to import from `__future__`?

Reply:
Otherwise anyone doing type checking in 3.9, it might not work?  This isn't my area of expertise, just questions from me.

Reply:
`from __future__ import annotations` is already at the top of the file.